### ISSUE TITLE
i have added my name

### ISSUE THREAD

**Issue by ARISHA2003** [GSSoC24/Postman-Challenge#1585]
pls approve the request

**Comment by ARISHA2003** [comment]
My name is not added yet

**Comment by MastanSayyad** [comment]
@ARISHA2003, you deleted your branch, which caused your PR to close automatically. Please create a new PR and make sure to delete the branch only after your PR has been merged.

**Comment by ARISHA2003** [comment]
Done
